server/views/pages/pause-notifications.ejs

```diff
@@ -0,0 +1,67 @@
+<%- include ('../helpers/helpers') -%>
```

Comment:
should really be writing new files in nunjucks instead of ejs, will have to convert them all at some stage